packages/webamp-modern-2/src/skin/ImageManager.ts

```diff
@@ -26,6 +39,84 @@ export default class ImageManager {
     return this._urlCache.get(filePath);
   }
 
+  // called only on RESOURCE_PHASE
+  // addBitmap(id:string, bitmap:Bitmap, filePath: string) {
+  addBitmap(bitmap:Bitmap) {
+    const id = bitmap._id;
+    const filePath = bitmap._file;
```

Comment:
Can we expose these (`_id` and `_file`) via some public method?

Reply:
done